lib/tasks/bower-install.js

```diff
@@ -37,32 +53,61 @@ class BowerInstallTask extends Task {
       .then(() => ui.writeLine(chalk.green('NPM: Installed bower')));
   }
 
+  get bowerJsonPath() {
+    return path.join(this.project.root, 'bower.json');
+  }
+
+  hasBowerJson() {
+    return existsSync(this.bowerJsonPath);
+  }
+
+  ensureBowerJson() {
+    if (this.hasBowerJson()) {
+      return Promise.resolve();
+    }
+
+    let projectName = this.project.name();
```

Comment:
vanity nit-pick: if this was `let name = this.project.name()` the `writeJson` bellow could use concise object syntax `{ name }`